src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/ManagedACTAccessRequirementWidgetViewImpl.java

```diff
@@ -369,10 +370,14 @@ public void showRequestAccessModal(
       List<JSONObjectAdapter> arList = new ArrayList<JSONObjectAdapter>();
       arList.add(arAsJson);
       AccessRequirementListProps.Callback onHide = () -> {
+        presenter.refreshApprovalState();
```

Comment:
The modal may hide when the user has submitted their request. It's easy enough to refresh the approval state in this callback, rather than adding a specific `onSuccess` callback.